src/fuel-toolchain.md

```diff
@@ -0,0 +1,31 @@
+# The Fuel Toolchain
+
+The full stack of tools designed and blessed by FuelLabs for enabling/assisting the Fuel application development experience.
+
+Tooling Overview:
+- `fuel-core`: The Fuel VM node client.
+- `forc`: The Fuel Orchestrator. Includes Sway compiler, packaging and plugin support.
+
+Official forc plugins:
```

Comment:
Not sure if we want to use the word "official" here. We want to avoid some notion of things being "official" since what's official or not is decided by the community. Fuel Labs is just one contributor among many to the Fuel protocol.